docs/docs/v7-upgrade-guidance.mdx

```diff
@@ -268,6 +268,14 @@ In version 7 of the CLI for Microsoft 365, we changed the `id` property of `spo
 
 Update your scripts to use the list item id for `id` property instead of UniqueId product id.
 
+## Enhanced `spo theme list`
```

Comment:
We didn't really enhance the command. Let's use something like: `Updated spo theme list command output`.